src/TextForEvent.tsx

```diff
@@ -669,6 +669,7 @@ export function hasText(ev): boolean {
     return Boolean(handler?.(ev));
 }
 
+export function textForEvent(ev: MatrixEvent): string
 export function textForEvent(ev: MatrixEvent, allowJSX = false): string | JSX.Element {
```

Comment:
![image](https://user-images.githubusercontent.com/2403652/123262173-81823b00-d4ef-11eb-8ffe-e62c4a4ccc9c.png)

My TS isn't happy with that 2nd arg

```suggestion
export function textForEvent(ev: MatrixEvent): string;
export function textForEvent(ev: MatrixEvent, allowJSX: true): string | JSX.Element;
export function textForEvent(ev: MatrixEvent, allowJSX = false): string | JSX.Element {
```

or similar